static/js/library/status.js

```diff
@@ -602,6 +613,47 @@ function update_metadata(event, elem, imdbid, tmdbid){
     });
 }
 
+function change_language(event, elem, imdbid, tmdbid){
```

Comment:
Codacy found an issue: ['change_language' is defined but never used.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)